### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by gdbjohnson** [Leafpub/leafpub#129]
Is there a Dockerfile out there for the Leafpub environment?  I can't really use the installation steps provided on my environment.

**Comment by karsasmus** [comment]
No, there isn't a Docker env. I've played a little bit with Docker and Leafpub but I haven't published yet the container setup
